tests/e2e/basic_test.go

```diff
@@ -156,9 +156,10 @@ func (s *BasicTestSuite) TestVmGroupVolumeIsolation(c *C) {
 	misc.LogTestStart(c.TestName())
 
 	// Initialize Config DB
-	admincli.ConfigInit(s.esx)
+	out, err := admincli.ConfigInit(s.esx)
+	c.Assert(err, IsNil, Commentf(out))
```

Comment:
I am totally agree for the separate testcase.

what is the harm of doing extra check here? It is a programatic step and helps to find out the root cause immediately instead of passing to next step and fail eventually with out proper cause.

It reduces the cost of extra debugging.

/CC @msterin